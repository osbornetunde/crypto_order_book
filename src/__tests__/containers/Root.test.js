import React from "react";
import { shallow } from "enzyme";
import Root from "../../containers/root";

describe("root component", () => {
  const storeWrapper = shallow(<Root />);
  it("expect root to render", () => {
    expect(storeWrapper).toMatchSnapshot();
  });

  it("checking the children of the root component", () => {
    expect(storeWrapper.children.length).toEqual(1);
  });
});
