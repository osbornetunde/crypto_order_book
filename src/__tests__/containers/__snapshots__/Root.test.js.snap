// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`root component expect root to render 1`] = `ShallowWrapper {}`;
